README.md

```diff
@@ -68,14 +82,57 @@ $ ./gk-deploy -g
 If you already have GlusterFS deployed in your cluster, you do not need the
 `-g` option.
 
-heketi should now be installed and ready to go. For ease of testing, we
-recommend the following:
+
+After this completes, to aid in testing and following the rest of the examples, the following export 
+should be run and will set the _HEKETI_CLI_SERVER_ env variable and will also give an
+easy means to echo the REST URL for the Heketi Server
 
 ```bash
 $ export HEKETI_CLI_SERVER=$(kubectl describe svc/heketi | grep "Endpoints:" | awk '{print "http://"$2}')
+
+$ echo $HEKETI_CLI_SERVER
+http://10.42.0.0:8080
+
+```
+
+
+kubernetes and heketi should now be installed and ready to go. 
+To verify Kubernetes installation fun the following commands:
+
+```bash
+$ kubectl get nodes
+NAME      STATUS    AGE
+master    Ready     22h
+node0     Ready     22h
+node1     Ready     22h
+node2     Ready     22h
 ```
 
-You should now be able to use `heketi-cli` to create volumes and then mount
+```bash
+$ kubectl get pods
+NAME                               READY     STATUS              RESTARTS   AGE
+glusterfs-node0-2509304327-vpce1   1/1       Running             0          1d
+glusterfs-node1-3290690057-hhq92   1/1       Running             0          1d
+glusterfs-node2-4072075787-okzjv   1/1       Running             0          1d
+heketi-3017632314-yyngh            1/1       Running             0          1d
+
+```
+
+Notice that we have 3 gluster pods running and 1 heketi pod running
+
+
```

Comment:
Similarly here, remove excessive newlines, and the two bash blocks can be merged together.